### PR TITLE
CLOUD-61260 - Reinstate recipes on Uluwatu UI

### PR DESCRIPTION
**Pull request by sodre90**
@keyki @doktoric pls review

### CODE REVIEW

File: core-api/src/main/java/com/sequenceiq/cloudbreak/validation/PluginValidator.java [hunk 1]
```diff
@@ -1,37 +1,36 @@
...
 
-import com.sequenceiq.cloudbreak.api.model.ExecutionType;
-
-public class PluginValidator implements ConstraintValidator<ValidPlugin, Map<String, ExecutionType>> {
+public class PluginValidator implements ConstraintValidator<ValidPlugin, Set<String>> {
 
     private static final String RECIPE_KEY_PREFIX = "consul-watch-plugin/";
-    private static final String URL_PATTERN = "^(http|https|git|consul|base64)://.*";
     private static final String CONSUL_PATTERN = "^consul://" + RECIPE_KEY_PREFIX + "([a-z][-a-z0-9]*[a-z0-9])";
+    private static final String URL_PATTERN = "^(consul|base64)://.*";
```
Comment (doktoric) [inline core-api/src/main/java/com/sequenceiq/cloudbreak/validation/PluginValidator.java]: @sodre90 do we need consul ?

Comment (sodre90) [inline core-api/src/main/java/com/sequenceiq/cloudbreak/validation/PluginValidator.java]: @keyki should I remove consul support?

Comment (keyki) [inline core-api/src/main/java/com/sequenceiq/cloudbreak/validation/PluginValidator.java]: Yes, please

Comment (sodre90) [inline core-api/src/main/java/com/sequenceiq/cloudbreak/validation/PluginValidator.java]: @keyki @doktoric done

**Pull request merged**